core/rio/turtle/src/main/java/org/eclipse/rdf4j/rio/turtle/TurtleWriter.java

```diff
@@ -108,6 +117,11 @@ public TurtleWriter(OutputStream out, ParsedIRI baseIRI) {
 		this.writer = new IndentingWriter(new OutputStreamWriter(out, StandardCharsets.UTF_8));
 	}
 
+	// TODO make injectable
+	private ModelFactory getModelFactory() {
+		return new LinkedHashModelFactory();
+	}
```

Comment:
You could just make it protected and then others could override this method if they have a better Model implementation they want to use.

Reply:
Forgot about this. I prefer making it properly injectable instead of forcing people to subclass - I'll add an optional constructor argument and also make it protected.